docs/source/extra/video-tutorials.rst

```diff
@@ -227,8 +227,6 @@ enable_docking() will enable experimental docking.
 
 By default, the user needs to hold the shift key to enable docking.
 
-The keyword shift_only = False enables docking without holding the shift key.
```

Comment:
Not sure why we're deleting this phrase but otherwise leaving this section unchanged... The phrase is as outdated as everything else here. Might be better not to touch it at all, lest it turns to ashes and gets blown away.